exercises/ex1.cpp

```diff
@@ -11,37 +11,32 @@ int main()
   cout << "Enter week number(1-7): " << endl;
   cin >> week;
 
-  if (week == 1)
+  switch (week)
```

Comment:
dovevi fare una branch separata per questo esercizio

Reply:
L'avevo fatta, non so perché non appena ho fatto la push mi ha inserito anche altri file, non vorrei che per sbaglio ho usato qualche comando in piu

Reply:
forse perchè l'hai fatta basandosi tulla branch dell'esercizio precedente, per basarti sulla "main" dovevi fare come da README (step 2)
git checkout main
git checkout -b nuova-branch